pkg/client/client_test.go

```diff
@@ -2238,7 +2238,8 @@ func TestPollUntil(t *testing.T) {
 
 	// the parent context times out.
 	var lastErr6 error
-	parentCtx, _ := context.WithTimeout(context.Background(), 10*time.Millisecond)
+	parentCtx, parentCancel := context.WithTimeout(context.Background(), 10*time.Millisecond)
+	defer parentCancel()
```

Comment:
PTAL https://github.com/openshift/cluster-monitoring-operator/pull/2234/files#diff-2dad64e28ce5e2eddf0adf303052d38e3b5b4fcba6b32fc4ca26d5e4d6b8fa4aR2373 (fails `go vet ./...`).

Reply:
Moved this (and the fix) to https://github.com/openshift/cluster-monitoring-operator/pull/2292.